libs/oidc/common/src/lib/entities/all.entity.ts

```diff
@@ -937,6 +937,20 @@ export class Role extends GuidIdentity {
 @Entity({
   name: 'user_roles'
 })
+export class NewUserRole extends GuidIdentity {
```

Comment:
Because this is will require delving even further into `oidc-provider-nest` if I change the original UserRole. I'm fine doing it, but I just want you to know that the PR will be EVEN BIGGER if I do that. You either deal with it now and I fix it for next PR (reimplementing 2FA, logging, pw reset, etc on IdP) or I do it now and create a bigger mess.